apps/dcos_l4lb/src/dcos_l4lb_mesos_poller.erl

```diff
@@ -49,47 +49,64 @@ start_link() ->
 %%%===================================================================
 
 init([]) ->
-    TRef = start_poll_timer(),
-    {ok, #state{timer_ref=TRef}}.
-
-handle_call(_Request, _From, State) ->
+    PollInterval = dcos_l4lb_config:agent_poll_interval(),
+    PollMaxInterval = dcos_l4lb_config:agent_poll_max_interval(),
+    Backoff0 = backoff:init(PollInterval, PollMaxInterval, self(), poll),
+    Backoff = backoff:type(Backoff0, jitter),
+    {ok, #state{backoff = Backoff}, {continue, poll}}.
+
+handle_call(Request, _From, State) ->
+    lager:warning("Unexpected request: ~p", [Request]),
     {reply, ok, State}.
 
-handle_cast(_Request, State) ->
+handle_cast(Request, State) ->
+    lager:warning("Unexpected request: ~p", [Request]),
+    {noreply, State}.
+
+handle_info({timeout, _TimerRef, poll}, State) ->
```

Comment:
Could you please store a reference in process state and explicitly match `TimerRef` as it was before your changes?